addons/project/views/project_sharing_project_task_views.xml

```diff
@@ -209,7 +209,10 @@
                             <field name="description" type="html" options="{'collaborative': true, 'allowCommandImage': false, 'allowCommandVideo': false, 'allowCommandFile': false}"/>
                         </page>
                         <page name="sub_tasks_page" string="Sub-tasks">
-                            <field name="child_ids" context="{'default_parent_id': id, 'default_partner_id': partner_id, 'form_view_ref' : 'project.project_sharing_project_task_view_form'}">
+                            <field name="child_ids" context="{'default_project_id': project_id, 'default_parent_id': id,
+                                'default_milestone_id': allow_milestones and milestone_id, 'default_partner_id': partner_id,
+                                'form_view_ref' : 'project.project_sharing_project_task_view_form',
+                            }">
```

Comment:
With the current implementation (by setting the default project_id) the newly created sub-task will show in the kanban view which is not desirable I believe. 
Is it not possible to add this ``'default_display_in_project': False,`` to the context like it's done in the "back-end" version of the project task form view.

Reply:
Done! :100: